content/reference/v1beta1/helmchart.md

```diff
@@ -30,6 +32,9 @@ spec:
   # helmVersion identifies the Helm Version used to render the Chart. Default is v2.
   helmVersion: v2
 
+  # useHelmInstall identifies whether this Helm chart will use the Replicated Helm installation (false) or native Helm installation (true). Default is false.
+  useHelmInstall: true
```

Comment:
We might have to rethink the wording here.  useHelmInstall: true is not compatible with helmVersion: v2.  Native helm must use helmVersion: v3.